docs/docs/06-transferring-props.md

```diff
@@ -143,7 +143,7 @@ z; // { a: 3, b: 4 }
 
 > Note:
 >
-> This proposal has reached stage 2 and is now enabled by default in Babel. Older versions of Babel may need to explicitly enable this transform with `babel --optional es7.objectRestSpread`
+> To transform rest and spread properties using Babel 6, you need to install [`es2015`](https://babeljs.io/docs/plugins/preset-es2015/) preset, [`transform-object-rest-spread`](https://babeljs.io/docs/plugins/transform-object-rest-spread/) plugin and configure them in the `.babelrc` file.
```

Comment:
This is fine but I'd like to tweak the wording slightly to include some definite articles.

> ... install *the* [`es2015`](https://babeljs.io/docs/plugins/preset-es2015/) preset,

And then the same for the plugin.

We'll make this easier though either by publishing our own preset or taking over the `react` preset so it includes this. In the mean time we should do this. We've just gone ahead and update our other docs for Babel 6 so I think we'll just continue down that route and not mention Babel 5 anymore.